.gitignore

```diff
@@ -1,3 +1,6 @@
+# Flask
+migrations/
```

Comment:
Hm, I don't think migrations folders should be gitignored, as once you do actually start using them, you want the folder, right? I have it checked into source control in my Flask apps.

Reply:
Found this comment in the Flask docs that confirms that: "This will add a migrations folder to your application. The contents of this folder need to be added to version control along with your other source files."
From:
https://flask-migrate.readthedocs.io/en/latest/index.html#example